### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mohanad-saqr**

### CODE REVIEW

File: sidebar.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/sidebar.css" />
+    <link rel="stylesheet" href="styles/main.css" />
```
Comment (karina-yuk) [inline sidebar.html:7]: This section of the code if you are linking to main.css it should be "...href="./assets/main.css" but if you are linking to sidebar.css it should come last and be "...href="./assets/sidebar.css".

File: sidebar.html
Comment (karina-yuk) [inline sidebar.html:1]: The webpage is not divided anymore and I don't see a sidebar section

**Comment by karina-yuk** [comment]
Please

**Review by karina-yuk — CHANGES_REQUESTED** [review]
The webpage is not divided anymore and I don't see a sidebar section

File: sidebar.html (hunk 1 above)
Comment (mohanad-saqr) [reply sidebar.html:7]: Good catch, I've made the change and committed.

File: sidebar.html [hunk 2]
```diff
@@ -4,9 +4,9 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles/sidebar.css" />
     <link rel="stylesheet" href="styles/main.css" />
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles/sidebar.css" />
```
Comment (karina-yuk) [inline sidebar.html:8]: The code here need to be change to "<link rel="stylesheet" href="./assets/sidebar.css"/>".

Comment (karina-yuk) [inline sidebar.html]: Change the code here to "...href="./assets/sidebar.css"...".

**Pull request closed without merging**